webapp/src/main/webapp/iaf/gui/js/controllers.js

```diff
@@ -1447,9 +1447,11 @@ angular.module('iaf.beheerconsole')
 			render: function ( data, type, row ) {
 				if(type === 'display'){
 					for(let i in data){
-						var columnData = data[i];
-						if(typeof columnData == 'string' && columnData.length > 30)
-							data[i] = '<span class="ellipsis" title="'+columnData+'">'+columnData.substr(0, 15)+' &#8230; '+columnData.substr(-15)+'</span>';
+						if(i !=='id'){
+							var columnData = data[i];
+							if(typeof columnData == 'string' && columnData.length > 30)
+								data[i] = '<span title="'+columnData.replace(/"/g, '&quot;')+'">'+columnData.substr(0, 15)+' &#8230; '+columnData.substr(-15)+'</span>';
+						}
```

Comment:
This should work without changing/adding a new column...?

Reply:
This changes/truncates the actual populated data in the table. Clicking View button uses this new data/id (<span ...) to fetch the from backend. To prevent this I created a new field messageId that will be truncated and the actual id will stay unchanged.